python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -416,6 +417,13 @@ def dataset_name(self) -> str:
         """Return the dataset name."""
         return self._dataset_name
 
+    @property
+    def full_node_set(self) -> Union[ItemSet, ItemSetDict]:
+        """Return the full node itemset."""
+        if self._full_node_set is None:
```

Comment:
Since we always load the `self._full_node_set` in `load` function. I believe there is no need to check if `self._full_node_set` is `None`.